conans/client/cmd/uploader.py

```diff
@@ -520,9 +520,9 @@ def compress_files(files, symlinks, name, dest_dir, output=None):
         if output and n_files > 1 and not output.is_terminal:
             output.write("[")
         elif output and n_files > 1 and output.is_terminal:
-            progress_bar = tqdm(total=len(files), desc="Compressing package...", 
-                                unit="files", leave=False, dynamic_ncols=True,
-                                ascii=False)
+            progress_bar = tqdm(total=len(files), desc="Compressing package...",
+                                unit="files", leave=True, dynamic_ncols=False,
+                                ascii=True)
```

Comment:
I think I should have put the output as an argument as well: `file=output` 
It makes all the progress bars look the same in width.
```suggestion
                                ascii=True, file=output)
```
line 553: `output.rewrite_line("{} [done]".format(progress_bar.desc))` should also be removed.